netket/experimental/driver/tdvp.py

```diff
@@ -173,6 +166,41 @@ def generator(self) -> Callable:
         """
         return self._generator
 
+    @property
+    def error_norm(self) -> Callable:
+        """
+        Returns the Callable function computing the error of the norm used for adaptive
+        timestepping by the integrator.
+        """
+        if self._error_norm == "euclidean_norm":
```

Comment:
Is it necessary to run the full logic here every time `error_norm` is called? It doesn't depend on anything that changes dynamically, if I'm not mistaken.

Reply:
We only call it once when constructing the integrator, no?

Reply:
otherwise yeah, i think you're right we're better off put this logic in the setter

Reply:
done